docs/kusion/3-concepts/3-module/3-app-dev-guide.md

```diff
@@ -9,7 +9,7 @@ To follow this guide, you will need:
 
 ## Workflow
 
-As a platform engineer, the workflow of developing a Kusion module looks like this:
+As an application developer, the workflow of developing a Kusion module looks like this:
```

Comment:
developing -> using ?

Reply:
My apologies for the oversight. It has been updated.